eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/AbstractHTTPServer.java

```diff
@@ -256,7 +294,18 @@ protected void channelRead0(ChannelHandlerContext ctx, HttpRequest httpRequest)
                 requestCommand.setHttpMethod(httpRequest.method().name());
                 requestCommand.setHttpVersion(httpRequest.protocolVersion().protocolName());
                 requestCommand.setRequestCode(requestCode);
-                // todo record command method, version and requestCode in span.
+
+                //todo record command opaque in span.
+                span = tracer.spanBuilder("HTTP "+requestCommand.httpMethod).setParent(context).setSpanKind(SpanKind.SERVER).startSpan();
+                span.addEvent("Span Start");
+                //attach the span to the server context
+                context = context.with(SpanKey.SERVER_KEY,span);
+                //put the context in channel
+                ctx.channel().attr(AttributeKeys.SERVER_CONTEXT).set(context);
+                //todo record command method, version and requestCode in span.
+                span.setAttribute("HttpMethod",httpRequest.method().name());
```

Comment:
ues otel key instaed : https://github.com/open-telemetry/opentelemetry-java/blob/main/semconv/src/main/java/io/opentelemetry/semconv/trace/attributes/SemanticAttributes.java

Reply:
I have no idea why I can't import this class.
![image](https://user-images.githubusercontent.com/84842773/135644953-135ffd17-e9a2-4c6f-aa8e-c644f42d5c51.png)
![image](https://user-images.githubusercontent.com/84842773/135645018-7fe5dfab-6c12-4afe-b71a-808caa127e0e.png)

Reply:
fixed